os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -90,6 +90,7 @@
 #ifdef CONFIG_ARMV7M_MPU
 #include "mpu.h"
 #endif
+#include <stdbool.h>
```

Comment:
bool is used only in your conditional. Let's add this inside of yours like stdio.h.

Reply:
bool is also used here,

https://github.com/pradeepns85/TizenRT/blob/d94b0d0e22244f3b64474bb8d91641f9df866bac/os/arch/arm/src/armv7-r/arm_assert.c#L112

Reply:
Oh ok. Thank you.